spotifyutils/__init__.py

```diff
@@ -51,6 +52,31 @@ def cli(input_args=None):
     args = parser.parse_args(input_args)
     main(**vars(args))
 
-
 def main(**kwargs):
-    print('Test')
+    """ Direct user to authorize spotify URL """
+
+    configfile = kwargs['configfile']
+    
+    config = configparser.ConfigParser()
+    config ['spotifyutils'] = {
+        'ClIENT_ID': kwargs['client_id'],
+        'CLIENT_SECRET': kwargs['client_secret'],
+        'REDIRECT_URI': kwargs['redirect_uri']
+    }
+
+    with open(configfile, "w") as f:
+        config.write(f)
+
+    parser = configparser.ConfigParser()
+    parser.read(configfile)
+
+    client_id = parser.get('spotifyutils', 'client_id')
+    client_secret = parser.get('spotifyutils', 'client_secret')
+    redirect_uri = parser.get('spotifyutils', 'redirect_uri')
+
+    print("Redirecting to Spotify Authorization URI, and spinning up web server")
```

Comment:
Most of this needs to be separated and wrapped in a specific "config" function, so that such an operation does not need to be performed each time.

Reply:
Wrapped this code in config function.

Reply:
This main method is the main entrypoint for all functionality. Authorization needs to be separated. There is a "config" subparser I thought. A lot of this config stuff belongs there.